cardinal/router/doc.go

```diff
@@ -0,0 +1,6 @@
+// Package router provides functionality for Cardinal to interact with the EVM Base Shard.
```

Comment:
hm, i would prefer to have this comment direclty on the struct definition or constructor so you get an explanation on it when you hover over the type def.

Reply:
removed to typedef